UI/ProductionWnd.cpp

```diff
@@ -723,17 +734,27 @@ namespace {
                 popup->AddMenuItem(GG::MenuItem(UserString("PAUSE"),            false, false, pause_action));
             }
 
-            if (queue_row && queue_row->elem.allowed_imperial_stockpile_use) {
-                popup->AddMenuItem(GG::MenuItem(UserString("DISALLOW_IMPERIAL_PP_STOCKPILE_USE"), false, false, disallow_stockpile_action));
-            } else {
-                popup->AddMenuItem(GG::MenuItem(UserString("ALLOW_IMPERIAL_PP_STOCKPILE_USE"), false, false, allow_stockpile_action));
+            switch (build_type) {
```

Comment:
Since there was a post-merge review: needs to catch other types/default, though it would probably read easier to replace the switch block with `if (build_type != BT_STOCKPILE) {`

Reply:
Lets move that discussion to https://github.com/freeorion/freeorion/pull/2041